plugins/knife/dns-records.rb

```diff
@@ -111,7 +111,12 @@ def delete_node_records(nodes)
     private
 
     def find_by_name_and_type(options)
-      zone.records.get(name(options), options[:type])
+      record = zone.records.get(name(options), options[:type])
+
+      # Sleep to comply with Route53 rate-limit
```

Comment:
I could, but I was trying to be consistent in always sleeping post performing a request against the EC2 APIs, rather than prior to performing a request.